src/closure_externs.js

```diff
@@ -92,10 +92,15 @@ var GlobalFetch;
 
 /**
  * @interface
- * @extends{WorkerGlobalScope}
- * @extends{Window}
+ * @extends {WorkerGlobalScope}
+ * @extends {Window}
  */
 var WindowOrWorkerGlobalScope;
 
 /** @record */
 function CanvasDrawImage() {}
+
+/** @constructor */
+function CryptoKey() {};
```

Comment:
Not sure I follow how this change is related to the rest of the PR?

Reply:
Sorry, I should have called that out. These are several commits in one pull request that are logically separate, but related. This change is required because the first change (better type emit) reveals a breakage where a user is missing this symbol, which is a builtin in TS but does not exist as an extern in JSCompiler.

Reply:
OK, makes sense. Thanks for clarifying.